config/ejabberd.exs

```diff
@@ -0,0 +1,169 @@
+defmodule Ejabberd.ConfigFile do
```

Comment:
I would add here:

```
import Ejabberd.Logger # this allow using info, error, etc for logging
```

Probably nice to add to the template, or even auto-import when you use `Ejabberd.Config`.